src/bitcoin-cli.cpp

```diff
@@ -414,14 +416,33 @@ static int CommandLineRPC(int argc, char *argv[])
             argc--;
             argv++;
         }
+        SetStdinEcho(false);
```

Comment:
- need to re-enable echoing before the normal `-stdin` processing
- make sure that this is not called unless either `-stdinrpcpass` or `-stdinwalletpassphrase` is called to avoid interfering with scripts that pipe in data, not attached to a TTY

Reply:
Good points. Addressed both.